app/views/layouts/finder_layout.html.erb

```diff
@@ -26,6 +26,5 @@
       </main>
       <%= render 'govuk_publishing_components/components/feedback' %>
     </div>
-    <%= render_mustache_templates unless Rails.env.production? %>
```

Comment:
Why did we have this in here? Do you know? Didn't we render the page the same as dev?

Reply:
This is mentioned in the installation section of the [shared_mustache readme](https://github.com/alphagov/shared_mustache/blob/bd11dbd0b1ccf700627ff12de8ddc2029d77bceb/README.md) - it's part of installing shared_mustache. No longer needed.